storm-client/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1559,8 +1577,8 @@ public static boolean isLocalhostAddress(String address) {
         Set<Integer> ids = srcMap.keySet();
         Integer largestId = ids.stream().max(Integer::compareTo).get();
         int end = largestId - start;
-        ArrayList<V> result = new ArrayList<>(Collections.nCopies(end+1 , null)); // creates array[largestId+1] filled with nulls
-        for( Map.Entry<Integer, V> entry : srcMap.entrySet() ) {
+        ArrayList<V> result = new ArrayList<>(Collections.nCopies(end + 1, null)); // creates array[largestId+1] filled with nulls
```

Comment:
Auto-corrected via IntelliJ. Let me revert if we don't want to fix unrelated thing.